pom.xml

```diff
@@ -252,7 +262,25 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-jdk14</artifactId>
         </dependency>
-        
+	<dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+	<dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jcl-over-slf4j</artifactId>
+        </dependency>
+	<dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jul-to-slf4j</artifactId>
+        </dependency>
```

Comment:
**question:** why do you include these three SLF4J libs as direct dependencies? 

`org.slf4j.slf4j-jdk14` has been included with a comment to have the lib on the classpath and serve as the logging bridge between SLF4J API and our common JUL logging. The others should IMHO not be necessary, as they are included as transitive dependencies by Tika and Solr4J.

Reply:
This may be true. I'm planning to take another pass at cleaning it all up before moving the PR into review officially.

Reply:
(also, I agree about adding an explicit "IllegalImport" rule for log4j)